sherpa/data.py

```diff
@@ -1745,7 +1765,7 @@ def html_data1d(data):
     if data.syserror is not None:
         meta.append(('Systematic error', data.syserror))
 
-    ls = [formatting.html_section(meta, summary=dtype + ' Summary',
+    ls = [formatting.html_section(meta, summary=f'{dtype} Summary',
```

Comment:
```suggestion
    ls = [formatting.html_section(meta, summary=f'{type(data).__name__} Summary',
```
and then then `dtype=...` line above can go.

Reply:
We do use `dtype` in line 1730 above so it already exists, so I don't see the benefit here.